dkam/pkg/script/uos/cpio_build/update_initramfs.sh

```diff
@@ -9,11 +9,11 @@ set -xueo pipefail
 data_dir=$1
 
 # shellcheck source=./secure_uos.sh
-source "$(dirname "$0")/secure_uos.sh"
+source secure_uos.sh
 
 pushd ../
 # shellcheck source=../config
-source "$(dirname "$0")/../config"
+source ./config
```

Comment:
Similarly, changed to source ./config. Same concern as above regarding the working directory.